src/api/v2/paths/directory.js

```diff
@@ -40,6 +35,9 @@ const get = async (req, res) => {
       if (ldeMessage.includes('Size Limit Exceeded')) {
         return errorBuilder(res, 400, ['Size Limit Exceeded (search too broad)']);
       }
+      if (ldeMessage.includes('Time Limit Exceeded')) {
+        return errorBuilder(res, 400, ['Query is too broad, causing a timeout on the database end. Please limit query to be more specific']);
```

Comment:
Is this the only reason that would cause a timeout? I think the message could be confusing if other issues caused the timeout

Reply:
I'll update the wording here to make more sense if the request times out for another reason